ovos_dinkum_listener/service.py

```diff
@@ -135,18 +138,15 @@ def __init__(self, on_ready=on_ready, on_error=on_error,
         self._before_start()  # connect to bus
         listener = self.config["listener"]
 
-        self.mic = mic or AlsaMicrophone(
-            device=listener.get("device_name") or "default",
-            sample_rate=listener.get("sample_rate", 1600),
-            sample_width=listener.get("sample_width", 2),
-            sample_channels=listener.get("sample_channels", 1),
-            chunk_size=listener.get("chunk_size", 4096),
-            period_size=listener.get("period_size", 1024),
-            multiplier=listener.get("multiplier", 1),
-            timeout=listener.get("audio_timeout", 5),
-            audio_retries=listener.get("audio_retries", 3),
-            audio_retry_delay=listener.get("audio_retry_delay", 1),
-        )
+        # Initialize with default (bundled) plugin
+        microphone_config = self.config.get("microphone", {})
+        microphone_config.setdefault('module', 'ovos-microphone-plugin-alsa')
```

Comment:
should use https://github.com/OpenVoiceOS/ovos-microphone-plugin-arecord if it's validated as working

pyaalsaaudio sometimes errors out installing, don't remember the details, either cython or swig or someother package needed at buildtime

arecord plugin has 0 python dependencies

Reply:
Get the following exception when using arecord:
```
2023-05-22 11:54:00.752 - voice - ovos_dinkum_listener.service:run:210 - ERROR - Service failed to start
Traceback (most recent call last):
  File "/home/neon/venv/lib/python3.7/site-packages/ovos_dinkum_listener/service.py", line 201, in run
    self.voice_loop.run()
  File "/home/neon/venv/lib/python3.7/site-packages/ovos_dinkum_listener/voice_loop/voice_loop.py", line 171, in run
    assert chunk is not None, "No audio from microphone"
AssertionError: No audio from microphone
```